etl/steps/data/garden/forests/2024-05-08/ifl.meta.yml

```diff
@@ -0,0 +1,23 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Forests & Deforestation
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+tables:
+  ifl:
+    variables:
+      ifl_area:
+        title: Intact Forest Landscapes Area
+        unit: square kilometers
+        short_unit: km²
+        description_short: Area of [intact forest landscapes](#dod:intact-forest-landscapes). Intact forest landscapes are vast, continuous natural ecosystems without significant human disturbance or fragmentation.
```

Comment:
I find it a little odd that you have a DoD as well as an explicit explanation. Wouldn't it be enough to simply say:
(A) "Area of [intact forest landscapes](#dod:intact-forest-landscapes)."
or:
(B) "Area of [intact forest landscapes](#dod:intact-forest-landscapes), which are vast, continuous natural ecosystems without significant human disturbance or fragmentation."

Reply:
I am a little unsure about this, my understanding was that in cases where you can it is best to include a short explanation in the subtitle alongside a DoD. Mainly so that when users screenshot rather than download the chart, as is often the case, a brief explanation of the term is still on the chart. 

In this case, the DoD goes more in-depth than the information captured in the `description_short`.